### DISCUSSION TITLE
Against custom  image

### DISCUSSION THREAD

**Discussion by icegood** [AlbrechtL/openwrt-docker#12] [General]
Hello. I need custom debugging of openwrt core features. And therefore I need custom docker image against my own configurable openwrt build. How do I do this?
What obstacles I would have?

**Comment by AlbrechtL** [comment]
If I understood you correct, you want to use your own image instead of the official once, correct?

You can try to adapt the Dockerfile by setting the variable `OPENWRT_IMAGE` to your image, see https://github.com/AlbrechtL/openwrt-docker/blob/d61dcf512c42ada74028510c936449c34f8730c6/Dockerfile#L90 and build you own OpenWrt container image.

Please note that only `*generic-squashfs-combined.img` images are supported.
